manager/src/main/java/com/redhat/service/smartevents/manager/logging/ManagerLoggingJsonProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.redhat.service.smartevents.manager.logging;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.logmanager.ExtLogRecord;
+
+import com.redhat.service.smartevents.infra.logging.CustomerIdProvider;
+
+import io.quarkiverse.loggingjson.JsonGenerator;
+import io.quarkiverse.loggingjson.JsonProvider;
+
+@ApplicationScoped
+public class ManagerLoggingJsonProvider implements JsonProvider {
+
+    private static final String WORKER_ID_JSON_KEY = "workerId";
+
+    @ConfigProperty(name = "event-bridge.resources.workers.id")
```

Comment:
@manstis I think we can just inject this in the `application.properties` file now, with the approach that @r00ta highlighted. Something like:

```
quarkus.log.console.json.additional-field.workderId.value=${event-bridge.resources.workers.id}
```

Reply:
Done.. Good catch. I missed that one.